src/backend/marsha/core/serializers/base.py

```diff
@@ -37,26 +37,41 @@
 KEY_REGEX = re.compile(KEY_PATTERN)
 
 
-def get_video_cloudfront_url_params(video_id):
+def get_resource_cloudfront_url_params(resource_kind: str, resource_id: str):
```

Comment:
Maybe we could match what you did for models :
```
S3_IDENTIFIER = "video"
```
```suggestion
def get_resource_cloudfront_url_params(s3_identifier: str, resource_id: str):
```
or 

```suggestion
def get_resource_cloudfront_url_params(resource_s3_identifier: str, resource_id: str):
```
or, even simpler use `resource_identifier` for both ?

Reply:
oh, I though it was related, please ignore my comments.